nikola/post.py

```diff
@@ -307,7 +305,6 @@ def __init__(
                     self.has_oldstyle_metadata_tags = True
 
         # While draft comes from the tags, it's not really a tag
```

Comment:
I think that’s a leftover comment from before the `status` meta key?

Reply:
Probably. Have not gotten that far yet